src/api/ledger/balances.js

```diff
@@ -31,14 +34,26 @@ function getTrustlinesAsync(account, options, callback) {
     .catch(callback);
 }
 
+function getLedgerVersion(remote: Remote, optionValue?: number,
+  callback: GetLedgerSequenceCallback
+) {
+  if (typeof optionValue === 'number') {
+    callback(null, optionValue);
+  } else {
+    remote.getLedgerSequence(callback);
+  }
+}
+
+
 function getBalancesAsync(account, options, callback) {
   validate.address(account);
   validate.getBalancesOptions(options);
 
-  const ledgerVersion = options.ledgerVersion
-                      || this.remote.getLedgerSequence();
   async.parallel({
-    xrp: _.partial(utils.getXRPBalance, this.remote, account, ledgerVersion),
+    xrp: async.compose(
```

Comment:
I think `async.seq` is a bit more intuitive since it orders the functions in execution order.